app/controllers/v2/event_invitations_controller.rb

```diff
@@ -21,14 +21,49 @@ def create
 
     def send_notifications(event_invitation)
       event_invitation.email_addresses_to_array.each do |email_address|
-        EventInvitationMailer.invite(
-          email_address: email_address,
-          event: event_invitation.event,
-          person: Person.find_by(email_address: email_address)
-        ).deliver_later
+        person = Person.find_by(email_address: email_address)
+
+        case person.preferred_contact_method
```

Comment:
we should maybe change this to `case person.preferred_contact_method.upcase` to make it more robust